app/command_autocomplete.go

```diff
@@ -174,19 +174,35 @@ func (a *App) parseArgument(commandArgs *model.CommandArgs, arg *model.Autocompl
 		}
 		parsed = changedParsed
 		toBeParsed = changedToBeParsed
+	} else if arg.Type == model.AutocompleteArgTypeBool {
+		found, changedParsed, changedToBeParsed, boolsuggestions := parseBoolArgument(arg, parsed, toBeParsed)
+		if found {
+			suggestions = append(suggestions, boolsuggestions)
+			return true, changedParsed, changedToBeParsed, suggestions
+		}
+		parsed = changedParsed
+		toBeParsed = changedToBeParsed
 	}
 
 	return false, parsed, toBeParsed, suggestions
 }
 
 func parseNamedArgument(arg *model.AutocompleteArg, parsed, toBeParsed string) (found bool, alreadyParsed string, yetToBeParsed string, suggestion model.AutocompleteSuggestion) {
 	in := strings.TrimPrefix(toBeParsed, " ")
+	if len(arg.Name) == 0 {
```

Comment:
Named argument can be can be a text argument or list argument as well as boolean. With this implementation every named argument will be autocompleted with `true` or `false`. What you have to do is check if we have bool named argument on our hand.

Reply:
sure i will have check named argument is bool type then only i will add the autocomplete true or false